client/src/pages/VotePage.tsx

```diff
@@ -59,6 +65,13 @@ export const VotePage = () => {
         setSelectionOption("");
         if (!isFocus) {
           document.title = questionStarted;
+          if (notifPermission === "granted") {
```

Comment:
Not all students will necessarily want this option all the time, so I would at least this as an icon that students can toggle (on the vote page)